<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageUI</class>
 <widget class="QMainWindow" name="MessageUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="2" column="0">
       <widget class="QTextEdit" name="SendTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>字体...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>表情</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="CleanPushButton">
          <property name="text">
           <string>清屏(&amp;R)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QTextEdit" name="ReceiveTextEdit"/>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>消息记录(&amp;H)...</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>关闭(&amp;C)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SendMessagePushButton">
          <property name="text">
           <string>发送(&amp;S)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>15</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>↓</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>250</height>
           </size>
          </property>
          <property name="text">
           <string>照片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>618</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>消息(&amp;M)</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_E"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>查看联系人资料(&amp;V)</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>关闭对话(&amp;C)</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>拷贝(&amp;C)</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>全选(&amp;A)</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>清屏(&amp;R)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
